activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/RuntimeTestConfiguration.java

```diff
@@ -34,27 +34,27 @@
 
 
     @Bean
-    public UserDetailsService myUserDetailsService() {
-        InMemoryUserDetailsManager inMemoryUserDetailsManager = new InMemoryUserDetailsManager();
+    public UserDetails myUserDetailsService() {
```

Comment:
@pancudaniel7 this is still wrong.. you need to use UserDetailsService you cannot cast to UserDetails.